src/Apcu/bootstrap.php

```diff
@@ -26,3 +26,13 @@ function apcu_inc($key, $step = 1, &$success = false) { return apc_inc($key, $st
     function apcu_sma_info($limited = false) { return apc_sma_info($limited); }
     function apcu_store($key, $var = null, $ttl = 0) { return apc_store($key, $var, $ttl); }
 }
+
+if (!class_exists('APCUIterator', false)) {
```

Comment:
I though that the existing check in https://github.com/symfony/polyfill/pull/40/files#diff-af8f2afac0a67b224b25386b2a75cc16R12 would be enough.
I don't get why, but it looks like so... hhvm?